mlir/test/Conversion/EmulateFp8ExtTrunc/Integration/roundtrip.mlir

```diff
@@ -36,6 +48,10 @@ func.func @printFail(%wanted : i8, %got : i8, %intermediate : f32) {
   call @printI64(%gotExt) : (i64) -> ()
   call @printComma() : () -> ()
   call @printF32(%intermediate) : (f32) -> ()
+  %interbits = arith.bitcast %intermediate : f32 to i32
```

Comment:
Print the intermediate float as bits -- NaNs were printing as "nan" and I needed to see exactly what was there.  Might as well keep the addition.